service/history/shard/context.go

```diff
@@ -624,67 +619,55 @@ func (s *contextImpl) CreateWorkflowExecution(
 		return nil, err
 	}
 
-Create_Loop:
-	for attempt := 0; attempt < conditionalRetryCount && ctx.Err() == nil; attempt++ {
-		if s.isClosed() {
-			return nil, ErrShardClosed
-		}
-		currentRangeID := s.getRangeID()
-		request.RangeID = currentRangeID
-
-		response, err := s.executionManager.CreateWorkflowExecution(ctx, request)
-		switch err.(type) {
-		case nil:
-			// Update MaxReadLevel if write to DB succeeds
-			s.updateMaxReadLevelLocked(transferMaxReadLevel)
-			return response, nil
-		case *types.WorkflowExecutionAlreadyStartedError,
-			*persistence.WorkflowExecutionAlreadyStartedError,
-			*persistence.CurrentWorkflowConditionFailedError,
-			*types.ServiceBusyError:
-			// No special handling required for these errors
-			// We know write to DB fails if these errors are returned
+	if s.isClosed() {
+		return nil, ErrShardClosed
+	}
```

Comment:
I'm guessing `s.isClosed()` is no longer possible now?

Reply:
I need to double confirm. Probably not needed anymore now